tests/DCPS/UnregisterType/run_test.pl

```diff
@@ -0,0 +1,37 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+    & eval 'exec perl -S $0 $argv:q'
+    if 0;
+
+# -*- perl -*-
+
+use Env (DDS_ROOT);
+use lib "$DDS_ROOT/bin";
+use Env (ACE_ROOT);
+use lib "$ACE_ROOT/bin";
+use PerlDDS::Run_Test;
+
+$status = 0;
+
+my $options = '';
+my $DCPSREPO;
+
+# The test results in error output from attempting to register
+# more instances than allowed. Capture it to prevent false alarms.
+my $logfile = 'output.log';
+
+unlink $logfile;
+
+$options = "-DCPSConfigFile rtps_disc.ini -ORBLogFile $logfile";
```

Comment:
The test shouldn't be hard-coded to use an RTPS Discovery configuration.  Since the test is independent of discovery mechanism, just use the defaults.  The problem with RTPS here is that it requires multicasting which can interfere with other builds unless certain steps are taken.  That is why tests that use RTPS as marked with an RTPS tag in dcps_tests.lst.